applications/smart-distancing/smart_distancing/core/_cv_distancing.py

```diff
@@ -26,18 +26,35 @@ def __init__(self, config):
         self.frame_counter = itertools.count(0)
         self.tracker = CentroidTracker(
             max_disappeared=int(self.config.get_section_dict("PostProcessor")["MaxTrackFrame"]))
+
+        detector_name_tmp = self.config.get_section_dict("Detector")["Name"]
         if self.device == 'Jetson':
-            from smart_distancing.detectors.jetson import MobilenetSsdDetector
-            self.detector = MobilenetSsdDetector(self.config)
+            if detector_name_tmp == 'mobilenet_ssd_v2_coco':
```

Comment:
These should be listed somewhere, I still have inner issues with derivations of BaseDetector class and the role of "Name" in the detector's config.